VideoEncoding/Encoding_H264_ContentAware_Constrained/README.md

```diff
@@ -72,115 +72,24 @@ To learn more about Content Aware Encoding, see the article on [Content Aware En
 1. Creates an input asset and upload a media file into it.
 1. Submits a job and monitoring the job using polling method.
 1. Downloads the output asset.
-1. prints urls for streaming.
-
-> [!TIP]
-> The `Program.cs` file has extensive comments.
+1. Prints URLs for streaming.
```

Comment:
The C# code provided in the readme file must be updated. The class name changed from PresetConfigurations( to EncoderPresetConfigurations {} and parameters have now uppercase etc....

Reply:
I've updated the code to resolve this issue.